books/RayTracingInOneWeekend.html

```diff
@@ -577,18 +626,29 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
         auto aspect_ratio = 16.0 / 9.0;
         int image_width = 400;
+
+        // Calculate the image height, and ensure that it's at least 1.
         int image_height = static_cast<int>(image_width / aspect_ratio);
+        image_height = (image_height < 1) ? 1 : image_height;
 
         // Camera
 
-        auto viewport_height = 2.0;
-        auto viewport_width = aspect_ratio * viewport_height;
         auto focal_length = 1.0;
+        auto viewport_height = 2.0;
+        auto viewport_width = viewport_height * (static_cast<double>(image_width)/image_height);
+        auto center = point3(0, 0, 0);  // Camera center
```

Comment:
why not just name it `camera_center`?

Reply:
I liked the succinctness of just `camera` in the code that uses it. And I'd have to rewrap some lines that use it. And the `sphere` class has a member `center`, which is a `<property>` of the `<thing>`.

There — that's all my ammunition.

I'm open to renaming to `camera_center` if you still want; let me know.

Reply:
While it's in main have it be `camera_center`. Once inside of it's own class rename to `center`?

Reply:
I like that; will do.